.builder/action/aws-c-http-test.sh

```diff
@@ -15,5 +15,10 @@ elif [ -d "../../aws-c-http" ]; then
     cd ../../aws-c-http
 fi
 
-ctest --output-on-failure
-
+LOCAL_HOST_SETUP=/usr/share/keyrings/nginx-archive-keyring.gpg
+if [ -f "$LOCAL_HOST_SETUP" ]; then
+    echo "run test with local host only"
+    ctest --output-on-failure -R hpack_stress
+else
+    ctest --output-on-failure
+fi
```

Comment:
let's either:
1) run all the tests all the time anyway
2) have an prefix like "integ_" that all the integration tests have, so we don't need to keep updating this list here